src/components/Comments.jsx

```diff
@@ -46,14 +48,37 @@ const Comments = (props) =>{
                     return updatedArticle;
                 })
                 setInput('');
+
             })
                 .catch((err) => {
                     alert(err)
+                    setInput('');
                     console.log(err);
                     setStatusBase({ msg: { err }, key: Math.random() });
                 })
         }
     }
+
+    const deleteHandler = (commentid) => {
+        deleteComment(commentid)
+            .then((res) => {
+                setComments((currcomments) => {
+                    const newComments = currcomments.filter((comment) => {
+                        return comment.comment_id !== commentid
+                    });
+                    return newComments;
+                })
+                props.setArticle((currArticle) => {
+                    const updatedArticle = { ...currArticle };
+                    updatedArticle['comment_count']--;
+                    return updatedArticle;
+                })
+            })
```

Comment:
Currently, I can delete all the comments regardless to who posted it. You should only be able to delete comments for the user that is logged in. This is a good time to put in a useContext for either a hard coded user logged in or a login feature where a user can choose who to log in as